apps/src/templates/certificates/Congrats.jsx

```diff
@@ -49,6 +55,49 @@ export default function Congrats(props) {
     );
   };
 
+  /**
```

Comment:
This is not a blocking comment, but have you considered moving the Extra certificates to a separate component?
Iw would make testing easier, and the code more readable.

Reply:
Yeah I have thought about it. Unfortunately I don't have time to do that refactoring right now :(